index.html

```diff
@@ -1,19 +1,91 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+  <body>
+    <header class="header">
+      <img src="img/karma-logo.svg" alt="Karma Logo" class="nav-logo" />
 
-</body>
+      <nav>
+        <ul class="nav-list">
+          <li class="nav-item-meet-karma">
+            <a class="nav-link-meet-karma" href="#">Meet Karma</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">How it Works</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Store</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Blog</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Help</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="">Login</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="background-image">
+        <div class="karma">
+          <h1>Introducing Karma</h1>
+          <p>Bring WiFi with you, everywhere you go.</p>
+          <button class="button">Learn More</button>
+        </div>
+      </section>
+
+      <section class="details">
+        <h1>Everyone needs a little Karma.</h1>
+        <div class="article-container">
```

Comment:
Articles are very close to each other - check how they look like on the target design (png image)

Reply:
I updated article container paddings 29 to 23rem.